### PR TITLE
[Merged by Bors] - docs: split up usage page and improve landing page

### PR DESCRIPTION
**Pull request by fhennig**
# Description

*Please add a description here. This will become the commit message of the merge request later.*

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/hbase/pages/usage-guide/logging.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      logging:
+        enableVectorAgent: true
+  regionServers:
+    config:
+      logging:
+        enableVectorAgent: true
+  restServers:
+    config:
+      logging:
+        enableVectorAgent: true
```
Comment (maltesander) [inline docs/modules/hbase/pages/usage-guide/logging.adoc:21]:
```suggestion
spec:
  clusterConfig:
    vectorAggregatorConfigMapName: vector-aggregator-discovery
  masters:
    config:
      logging:
        enableVectorAgent: true
  regionServers:
    config:
      logging:
        enableVectorAgent: true
  restServers:
    config:
      logging:
        enableVectorAgent: true
```

File: docs/modules/hbase/pages/usage-guide/overrides.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+IMPORTANT: Overriding certain properties which are set by operator can interfere with the operator and can lead to problems.
+
+== Configuration Properties
+
+For a role or role group, at the same level of `config`, you can specify: `configOverrides` for the following files:
+
+- `hbase-site.xml`
+- `hbase-env.sh`
+
+// hdfs-site.xml is not listed here. The file is always taken from the referenced hdfs cluster
```
Comment (maltesander) [inline docs/modules/hbase/pages/usage-guide/overrides.adoc:15]: Should we mention this not just in a comment?

**Review by maltesander — CHANGES_REQUESTED** [review]
Just minor things.

File: docs/modules/hbase/pages/usage-guide/overrides.adoc (hunk 2 above)
Comment (fhennig) [reply docs/modules/hbase/pages/usage-guide/overrides.adoc:15]: Good idea

Comment (fhennig) [reply docs/modules/hbase/pages/usage-guide/overrides.adoc]: @maltesander I've added a note in [67f7685](https://github.com/stackabletech/hbase-operator/pull/344/commits/67f7685861f6f631c10c72e77cb3da859ab09dc5)

**Review by maltesander — APPROVED** [review]
LGTM!

**Comment by fhennig** [comment]
bors merge

**Pull request closed without merging**